src/main/drivers/max7456.c

```diff
@@ -212,7 +212,7 @@ static uint8_t shadowBuffer[VIDEO_BUFFER_CHARS_PAL];
 volatile bool dmaTransactionInProgress = false;
 #endif
 
-static uint8_t spiBuff[MAX_CHARS2UPDATE*6];
+static uint8_t spiBuff[MAX_CHARS2UPDATE*6+40]; // Took the extra 40 bytes from screenBuffer
```

Comment:
From comment it seems that some code assumes that there is extra space in screenBuffer. No change in this PR seems to change this, so it will probably result in overflow somewhere.

Reply:
#8063 fixed the overflow. the extra space is no longer needed. Old old code for clearing screenBuffer was other vector of overflow.